dist/tools/cpy2remed/cpy2remed.sh

```diff
@@ -10,4 +16,41 @@ DEV_DIR="$2"
 
 REMED_MOUNT_PATH="${REMED_MOUNT_PATH:-/media/$USER/$DEV_DIR/}"
 
-cp "$HEXFILE" "$REMED_MOUNT_PATH"
+if test -e $REMED_MOUNT_PATH
```

Comment:
```suggestion
if [ -d "$REMED_MOUNT_PATH" ]
```

Reply:
Hmmm ... I looked previously in the output of static test (at the page bottom) and everything was green. How can I go to the output with this static test warnings?